app/views/katello/api/v2/repositories/base.json.rabl

```diff
@@ -25,6 +25,16 @@ node :content_label do |repo|
   repo.content.try(:label)
 end
 
+child :published_in_versions => :versions do |version|
```

Comment:
What do you think about naming this `content_view_versions` instead? 'versions' without any context makes me think "repository versions" which obviously isn't right. may be just me though.